configs/13.0/packages/python/sources

```diff
@@ -20,17 +20,17 @@
 #
 
 ATSRC_PACKAGE_NAME="Python"
-ATSRC_PACKAGE_VER=3.7
-ATSRC_PACKAGE_REV=1bc6cd706670
+ATSRC_PACKAGE_VER=3.7.3
+ATSRC_PACKAGE_REV=ef4ec6ed12d6
```

Comment:
OK.  Although python provides stable[1] branches, such as the 3.7 branch, we only update between point releases.
```
$ git describe ef4ec6ed12d6
v3.7.3
```

[1] I'm not even sure if the 3.7 branch is supposed to be seen as a stable branch, or if it should be treated as a development branch for the 3.7 series.